presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -363,6 +363,44 @@ void PrestoServer::run() {
         json infoStateJson = convertNodeState(server->nodeState());
         http::sendOkResponse(downstream, infoStateJson);
       });
+  httpServer_->registerPut(
+    "/v1/info/state",
+    [server = this](
+        proxygen::HTTPMessage* /*message*/,
+        const std::vector<std::unique_ptr<folly::IOBuf>>& body ,
+        proxygen::ResponseHandler* downstream) {
+         std::string requestBody;
```

Comment:
Please  move this logic into its own function.

Reply:
Ok. I will create getRequestBody function for this.